client/src/main/scala/org/apache/celeborn/client/WorkerStatusTracker.scala

```diff
@@ -176,20 +184,22 @@ class WorkerStatusTracker(
               }
           }
       }
-      for (worker <- res.excludedWorkers.asScala) {
-        if (!excludedWorkers.containsKey(worker)) {
-          excludedWorkers.put(worker, (StatusCode.WORKER_EXCLUDED, current))
+      for (workerSummary <- res.excludedWorkers.asScala.map(WorkerId.fromWorkerInfo)) {
```

Comment:
can you replace all workerSummary with workerId? Otherwise, it looks confusing to developers.

Reply:
+1

Reply:
sure, the local variable is missed, will check and replace all

Reply:
done